content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -35,38 +37,36 @@ Users need to be aware of the following when using this feature:
 
 * API Objects `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass` are CRDs, not part of the core API.
 * `VolumeSnapshot` support is only available for CSI drivers.
-* As part of the deployment process, the Kubernetes team provides a sidecar helper container for the snapshot controller called `external-snapshotter`. It watches `VolumeSnapshot` objects and triggers `CreateSnapshot` and `DeleteSnapshot` operations against a CSI endpoint.
+* As part of the deployment process in `VolumeSnapshot` API v1beta1, the Kubernetes team provides two external snapshot controllers called `external-snapshotter`, a common controller to be deployed on the controller node and a sidecar helper container to be deployed together with the CSI driver.  The common controller watches `VolumeSnapshot` and `VolumeSnapshotContent` objects and is responsible for the creation and deletion of `VolumeSnapshotContent` object in dynamic provisioning.  The sidecar controller watches `VolumeSnapshotContent` objects and triggers `CreateSnapshot` and `DeleteSnapshot` operations against a CSI endpoint.
 * CSI drivers may or may not have implemented the volume snapshot functionality. The CSI drivers that have provided support for volume snapshot will likely use `external-snapshotter`.
-* The CSI drivers that support volume snapshot will automatically install CRDs defined for the volume snapshots.
+* The `VolumeSnapshot` v1beta CRDs need to be installed manually by the CSI drivers.
 
 ## Lifecycle of a volume snapshot and volume snapshot content
 
 `VolumeSnapshotContents` are resources in the cluster. `VolumeSnapshots` are requests for those resources. The interaction between `VolumeSnapshotContents` and `VolumeSnapshots` follow this lifecycle:
 
 ### Provisioning Volume Snapshot
 
-There are two ways snapshots may be provisioned: statically or dynamically.
+There are two ways snapshots may be provisioned: pre-provisioned or dynamically provisioned.
 
-#### Static
+#### Pre-provisioned
```

Comment:
Optional tweak:
```suggestion
#### Pre-provisioned {#static}
```
(if there are existing inbound hyperlinks, this change means those existing links will stay working).